src/libultra/io/pfsfreeblocks.c

```diff
@@ -1,3 +1,37 @@
 #include "global.h"
+#include "PR/pfs.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/pfsfreeblocks/osPfsFreeBlocks.s")
+s32 osPfsFreeBlocks(OSPfs* pfs, s32* leftoverBytes) {
```

Comment:
```suggestion
s32 osPfsFreeBlocks(OSPfs* pfs, size_t* leftoverBytes) {
```

Reply:
Notably, the only time this function is used, in `osPfsAllocateFile`, the argument has to be `s32` to match:
![image](https://user-images.githubusercontent.com/73679967/146852858-ab47c310-c29f-4906-bdfe-336de469ce24.png)